### ISSUE TITLE
Handling of data

### ISSUE THREAD

**Issue by sc0rp10n-py** [kaarana/CoWIN-Data#3]
Is there any reason for keeping data for each day? Why not just keep one file and update the data of file at end of each day?

**Comment by srikanthlogic** [comment]
For a time series visualization, daily data helps. I am seeing this more as a archival project

**Comment by aflah02** [comment]
Makes sense also @srikanthlogic it seems when i run the cowin scripts right now it loads all the data into the main folder instead of the data folder is that intentional for some reason?

**Comment by sc0rp10n-py** [comment]
> For a time series visualization, daily data helps. I am seeing this more as a archival project

Ohk, if it's archival then it makes sense. I am working on its front end part, so for that as it's archival just simple data in table form with graphs would work. Right?
Anything else I should consider?
